test/verify/composerlib.py

```diff
@@ -52,6 +52,11 @@ def setUp(self):
                         "for bp in $(composer-cli blueprints list); "
                         "do composer-cli blueprints delete $bp; done")
 
+    # Hack to add more wait time to work with aarch64 platform test
+    def login_and_go(self, *args, **kwargs):
+        with self.browser.wait_timeout(300):
```

Comment:
This could use 300 only on aarch64, and otherwise keep the default, so that a failing test case doesn't regularly sit around for 5 minutes until it ends. But that's not important at all.